# Description: Link scripts in the same way Python3 does
# Submitted by: mva

--- Makefile.pre.in.orig	2020-04-19 21:13:39 UTC
+++ Makefile.pre.in
@@ -994,6 +994,12 @@ bininstall:	altbininstall
 	(cd $(DESTDIR)$(LIBPC); $(LN) -s python-$(VERSION).pc python2.pc)
 	-rm -f $(DESTDIR)$(LIBPC)/python.pc
 	(cd $(DESTDIR)$(LIBPC); $(LN) -s python2.pc python.pc)
+	-rm -f $(DESTDIR)$(BINDIR)/idle
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s idle$(VERSION) idle2)
+	-rm -f $(DESTDIR)$(BINDIR)/pydoc
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s pydoc$(VERSION) pydoc2)
+	-rm -f $(DESTDIR)$(BINDIR)/2to3
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s 2to3-$(VERSION) 2to3)
 
 # Install the interpreter with $(VERSION) affixed
 # This goes into $(exec_prefix)
